ci-operator/jobs/openshift/baremetal-runtimecfg/openshift-baremetal-runtimecfg-master-presubmits.yaml

```diff
@@ -1,5 +1,82 @@
 presubmits:
   openshift/baremetal-runtimecfg:
+  - agent: kubernetes
+    always_run: true
```

Comment:
Shouldn't this be:

```
always_run: false
```

Reply:
Huh. I wonder if that got changed when I ran make jobs. I could swear I set it to false originally.

Reply:
If activity on baremetal-runtimecfg isn't running tons of tests per day, it  should be fine to turn it on.